main.yml

```diff
@@ -344,6 +344,16 @@ migration-analytics:
       - /migrations/migration-analytics
   source_repo: https://github.com/project-xavier/xavier-ui
 
+my-user-access:
+  title: My user access
+  channel: '#platform-experience'
+  deployment_repo: https://github.com/RedHatInsights/insights-rbac-ui
```

Comment:
The deployment repo should probably be different than the source repo, unless this is a weird one